com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/ShaderVariablesLightLoop.hlsl

```diff
@@ -6,6 +6,7 @@
 StructuredBuffer<uint> _CoarseTileBuffer;
 StructuredBuffer<uint> _FineTileBuffer;
 StructuredBuffer<uint> _zBinBuffer;
+StructuredBuffer<uint> _zBinBitArrayBuffer;
```

Comment:
Note: Even if the code is cleaner by doing a separation between  the _zBinBuffer and the _zBinBitArrayBuffer 
in practice there is no real need to separate it. What matter is how you index into it, i.e ComputeZBinBufferIndex (where you index with variable category size) and how you interpret the data based on the bool depth sorted (i.e min max or a set of bit). This will allow to share memory (i.e we allocate a single buffer of uint and all the memory is shared). Or I miss something?